public/controllers/aeon_requests_controller.rb

```diff
@@ -146,7 +146,7 @@ def ancestry_titles_for(record)
         raise RecordNotFound.new if not node["publish"] == true
 
         if node["node_type"] === "resource"
-          breadcrumb_title = (node["finding_aid_status"] === 'completed' and not node["finding_aid_title"].nil?) ? node["finding_aid_title"] : node["title"]
+          breadcrumb_title = (node["finding_aid_status"] === 'completed' and not node["title"].nil?) ? node["title"] : node["title"]
```

Comment:
This is equivalent to: `breadcrumb_title = node["title"]` as `node["title"]` is going to be used regardless given the update. It looks like you want to prefer `node["title"]` even if "finding_aid_title" is defined and the status is completed?